charts/kube-prometheus-stack/templates/prometheus-operator/certmanager.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.prometheusOperator.admissionWebhooks.certManager.enabled -}}
+---
+# Create a selfsigned Issuer, in order to create a root CA certificate for
```

Comment:
Can you add an option to reuse an existing issuer?

Reply:
I've updated the PR so you can now specify an issuerRef value.